packages/api/src/routes/internal/index.ts

```diff
@@ -68,6 +68,7 @@ router.use("/hie", hieRoutes);
 router.use("/feedback", feedbackRoutes);
 router.use("/token", jwtToken);
 router.use("/ehr", ehr);
+router.use("/job", jobRoutes);
```

Comment:
I was torn between leaving as is and moving this under `/internal/patient`... what made me come back to it and suggest to move this under `patient` is that I think the job routes should have the job ID on the path, like:

```
/internal/patient/job/:id/update-totals
```

Reply:
I think you didn't understand what I meant :)

I mean having job after patient, not the other way around:

...this

```
POST /internal/patient/job/:id/update-total
```

...instead of this

```
POST /internal/job/patient/update-total/:jobId
```

The REST way of building paths, where we have `/entity` for general operations in the group of entities, and `/entity/:id` for operations against a single entity.